lib/arrays.md

```diff
@@ -5,44 +5,60 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An Array is a data object that stores an ordered list of values.
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+a collection of elements, each selected by one ore more identifying keys. 
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+puts arr[5] would get you the 6th element. Since arrays are zero-based indexed.
```

Comment:
I'm being a bit particular here: In the statement `puts arr[5]` which part of the expression is _retrieving an element_ from the Array? What is the other part of the expression responsible for doing?